technologies/job/aws-sagemaker/clone-trainingjob/instanceActions.js

```diff
@@ -0,0 +1,182 @@
+const AWS = require('aws-sdk');
+const PAKO = require('pako')
+const { Response, JobStatus, Log , Stream} = require('@saagie/sdk');
+
+/**
+ * Logic to start the external job instance.
+ * @param {Object} params
+ * @param {Object} params.job - Contains job data including featuresValues.
+ * @param {Object} params.instance - Contains instance data.
+ */
+exports.start = async ({ job, instance }) => {
+  try {
+    console.log('START INSTANCE:', instance);
+
+    if ( job.featuresValues.trainingjobname == null || job.featuresValues.trainingjobname.trim() === '' ) {
+          return Response.error('Training Job name is required.', new Error("Validation Error"));
+    }
+
+    if ( job.featuresValues.trainingjobinput == null || job.featuresValues.trainingjobinput.trim() === '' ) {
+          return Response.error('Training Job Input S3 location is required.', new Error("Validation Error"));
+    }
+
+    if ( job.featuresValues.trainingjoboutput == null || job.featuresValues.trainingjoboutput.trim() === '' ) {
+          return Response.error('Training Job Output S3 location is required.', new Error("Validation Error"));
+    }
+
+    AWS.config.update({credentials: { accessKeyId : job.featuresValues.endpoint.aws_access_key_id, secretAccessKey:  job.featuresValues.endpoint.aws_secret_access_key}});
+    AWS.config.update({region: job.featuresValues.endpoint.region});
+    
+    const sagemaker = new AWS.SageMaker({apiVersion: '2017-07-24'});
+    console.log('Get Job Details',  job.featuresValues.trainingjobs.label)
+    const data = await sagemaker.describeTrainingJob( params = {
+      TrainingJobName: job.featuresValues.trainingjobs.label
+    }).promise();
+
+    console.log("Retrieved Template Job Details");
+    console.log(data);
+
+    console.log("Create New Job based on template parameters");
+    var jparams = {
```

Comment:
```suggestion
    const jparams = {
```

https://github.com/saagie/sdk/wiki/Best-Practices#var-let-and-const-